Protocol.md

```diff
@@ -99,17 +70,73 @@ have no further frames sent and all frames will be ignored.
 
 ReactiveSocket as specified here only allows for TCP, WebSocket, and Aeron as transport protocols.
 
-#### Frame Length
+### Framing Protocol
+
+Some of the supported transport protocols for ReactiveSocket may not support specific framing of messages. For these protocols, a framing protocol must be
+used with the ReactiveSocket frame.
 
-The presence of the Frame Length field (and corresponding __R__ bit) is determined by the transport protocol being used. The frame length field MUST be omitted if the transport protocol preserves message boundaries e.g. provides compatible framing. If, however, the transport protocol only provides a stream abstraction or can merge messages without preserving boundaries, or multiple transport protocols may be used, then the frame length field MUST be used.
+The framing header MUST be omitted if the transport protocol preserves message boundaries e.g. provides compatible framing. If, however, the transport protocol only provides a stream abstraction or can merge messages without preserving boundaries, or multiple transport protocols may be used, then the frame header MUST be used.
 
 |  Transport Protocol            | Frame Length Field Required |
 |:-------------------------------|:----------------------------|
 | TCP                            | __YES__ |
 | WebSocket                      | __NO__  |
 | Aeron                          | __NO__  |
+| HTTP/2                         | __NO__  |
```

Comment:
This is from a pure framing perspective. Lots more detail of how to run over HTTP/2 would be needed. But framing is not one of them.